views/index.ejs

```diff
@@ -23,54 +23,19 @@
     <!--Deployed commit: <%= process.env.HEAD_COMMIT %>-->
     <!--Build env: <%= process.env.NODE_ENV %>-->
 
-    <noscript>
-      <p style="text-align: center;">
-        Sorry, it looks like you have JavaScript disabled.
-        <strong><a href="http://enable-javascript.com/">Learn how to enable it.</a></strong>
-      </p>
-    </noscript>
+    <h1>Under Maintenance</h1>
+    
+    <p>The Zooniverse is currently under maintenance. We'll be up again in a few hours.</p>
```

Comment:
You're correct, the migration is only supposed to take 10-20 minutes, but I put in a safety buffer just in case, to manage user expectations. To be fair though, 10-20 minutes => a few hours might be a wee bit paranoid.

Reply:
You know what, "a few hours" is overkill. I'm re-writing it to "We'll be up again in an hour" - that should be more reasonable _and_ provide us with a workable safety buffer.